src/style.css

```diff
@@ -1 +1,82 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
```

Comment:
Only box-sizing is allowed to pass into `* {}` selector

Reply:
Hey! You haven't resolve this comment.
Split your rules like this:
```
* {
  box-sizing: border-box;
}

body {
  margin: 0;
  font-family: "Roboto", sans-serif;
  font-weight: 500;
  font-size: 12px;
  line-height: 14px;
  text-transform: uppercase;
}
```